solutions/bronze/usaco-784.mdx

```diff
@@ -75,138 +73,209 @@ import java.util.*;
 import java.io.*;
 
 public class Lifeguards {
-    public static void main(String[] args) throws IOException {
-        PrintWriter pw = new PrintWriter(new File("lifeguards.out"));
-        BufferedReader br = new BufferedReader(new FileReader(new File("lifeguards.in")));
-        StringTokenizer st;
-
-        int cows = Integer.parseInt(br.readLine());
-        int[] startT = new int[cows];
-        int[] endT = new int[cows];
-        int[] times = new int[1000];
-        int shiftTotal = -1;
-
-        for (int c = 0; c < cows; c++){
-            st = new StringTokenizer(br.readLine());
-            startT[c] = Integer.parseInt(st.nextToken());
-            endT[c] = Integer.parseInt(st.nextToken());
-            for (int k = startT[c]; k < endT[c]; k++){
-                times[k]++;
-            }
-        }
-
-        for (int i = 0; i < cows; i++) {
-            for (int t = startT[i]; t < endT[i]; t++) {
-                times[t]--;
-            }
-
-            int covered = 0;
-            for(int t = 0; t < 1000; t++) {
-                if(times[t] > 0) {
-                    covered++;
-                }
-            }
-            shiftTotal = Math.max(shiftTotal, covered);
-
-            for (int t = startT[i]; t < endT[i]; t++) {
-                times[t]++;
-            }
-        }
-        pw.println(shiftTotal);
-        pw.close();
-        br.close();
-    }
+	public static void main(String[] args) throws IOException {
+		PrintWriter pw = new PrintWriter(new File("lifeguards.out"));
+		BufferedReader br = new BufferedReader(new FileReader(new File("lifeguards.in")));
+		StringTokenizer st;
+
+		int cows = Integer.parseInt(br.readLine());
+		int[] startT = new int[cows];
+		int[] endT = new int[cows];
+		int[] times = new int[1000];
+		int shiftTotal = -1;
+
+		for (int c = 0; c < cows; c++){
+			st = new StringTokenizer(br.readLine());
+			startT[c] = Integer.parseInt(st.nextToken());
+			endT[c] = Integer.parseInt(st.nextToken());
+			for (int k = startT[c]; k < endT[c]; k++){
```

Comment:
why k
since u use t in other places just use t lol

Reply:
this is part of the video solution...